Mathlib/LinearAlgebra/RootSystem/IsValuedIn.lean

```diff
@@ -255,6 +265,12 @@ lemma iInf_ker_coroot'_eq :
     span R (range P.coroot') = ⊤ :=
   span_root'_eq_top P.flip
 
+lemma pairingIn_zero_iff {S : Type*} [CommRing S] [Algebra S R] [FaithfulSMul S R]
```

Comment:
```suggestion
lemma pairingIn_eq_zero_iff {S : Type*} [CommRing S] [Algebra S R] [FaithfulSMul S R]
```

Reply:
Done. Also applied to [RootPairing.pairing_zero_iff](https://leanprover-community.github.io/mathlib4_docs/Mathlib/LinearAlgebra/RootSystem/Defs.html#RootPairing.pairing_zero_iff) for consistency.